Documentation/About.rst

```diff
@@ -0,0 +1,54 @@
+.. include:: Includes.txt
+
+
+.. _about:
+.. _about-this-document:
+
+===================
+About This Tutorial
+===================
+
+This document was created to introduce new users to TYPO3, its core features and
+provide an overview on how to configure and administer the CMS.
+
+Upon completing this tutorial, you should have a better understanding about how
+the CMS is installed, how the backend is administered and how you begin creating
+templates.
+
+French Translation
+==================
+
+A French translation has been created by Jonathan Iroulin.
+
+We are currently working on optimizing the rendering. Due to this, there
+was an issue with rendering the translation. The translated version still
+exists in a separate branch **fr** and only needs to be reactivated once
+the issues with rendering have been solved and the French branch has been
```

Comment:
Which rendering issues do we have? This should work now with new Bamboo Rendering pipeline.

Reply:
Yes, the note was added. The rendering issues have been resolved, I think.

However, the French translation should be reviewed for TYPO3 10 before putting it online again.